sklearn/utils/tests/test_testing.py

```diff
@@ -235,3 +238,129 @@ def f():
 
         if failed:
             raise AssertionError("wrong warning caught by assert_warn")
+
+
+# Tests for docstrings:
+
+def f_ok(a, b):
+    """Function f
+
+    Parameters
+    ----------
+    a : int
```

Comment:
A common mistake is that the colon is forgotten before the type. Can we check this case is handled well?

Numpydoc also allows the type to be missing altogether, but only without the colon. We should be checking this case too, perhaps, or using pydocstyle to lint some of these issues